Modules/Core/Controllers/MailerController.php

```
@@ -2,160 +2,187 @@
+     * @legacy-function invoice
+     * @legacy-file application/modules/mailer/controllers/Mailer.php
      */
     public function invoice(Request $request, int $invoice_id)
     {
-        if ( ! $this->mailer_configured) {
+        if (!$this->mailer_configured) {
             return;
         }
-        $invoice           = (new InvoicesService())->getById($invoice_id);
+
+        $invoice = $this->invoiceService->getById($invoice_id);
```

PHPMD (2.15.0)

The variable $invoice_id is not named in camelCase. (undefined)

(CamelCaseVariableName)